dockers/pipecd-base/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:326.0.0-alpine
```

Comment:
It will be better if we don't depend on this pre-built base image.
In the future, we may also need to install other things, for example, AWS tool to create indexes for DynamoDB.
So, just installing gcloud command tool in a pure alpine image looks good.